Text-Sentimental-Analysis-Scipt-with-GUI/text_sentimental_application.py

```diff
@@ -0,0 +1,47 @@
+import textSentimentalAnalysisWebApp.Model
+
+from tkinter import *
+from tkinter import messagebox
+
+
+def search():
+    word = enterWordEntry.get()
+
+    text = word.strip()
+    model_object = textSentimentalAnalysisWebApp.Model.model()
```

Comment:
```suggestion
    model_object = Model.model()
```

Reply:
please if all the suggested issues are resolved or not let me know if any more changes are required